src/gui/updownratiodlg.cpp

```diff
@@ -28,13 +28,16 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include <QMessageBox>
+
 #include "updownratiodlg.h"
```

Comment:
I wasn't gonna say anything because apparently you hate doing coding style but while you're at it reverse these: 
```
#include "updownratiodlg.h"

#include <QMessageBox>
```